tests/sys_crypto/tests-crypto-chacha20poly1305.c

```diff
@@ -21,18 +21,27 @@ uint8_t ebuf[1024];
 uint8_t pbuf[1024];
 
 static const uint8_t key_1[32] = {
-    0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b, 0x8c, 0x8d, 0x8e, 0x8f,
-    0x90, 0x91, 0x92, 0x93, 0x94, 0x95, 0x96, 0x97, 0x98, 0x99, 0x9a, 0x9b, 0x9c, 0x9d, 0x9e, 0x9f
+    0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b,
+    0x8c, 0x8d, 0x8e, 0x8f,
```

Comment:
Same issues with keys here. But this time, just revert the changes on keys, not the changes on the whole file :)
Use `git reset -p` or something similar.

Reply:
Sorry, I'm not gonna do manual code style changes to files I'm not even touching. 
Reverted: 1b5f42a9

Reply:
> I'm not gonna do manual code style changes to files I'm not even touching

Makes sense. Looks good now. Could you just squash your changes into a single commit ?

Reply:
Done.
Btw: Since when *sys/crypto*? The last PRs in this area I've done should always have been prefixed with just *crypto*.

Reply:
> Since when sys/crypto?

Because the changes are applied to files in the sys/crypto module. It would have been even better to put the changes in tests/ in a separate commit but I don't want to bother you much.